sqlstore/src/main/resources/db/migration_afterMigrate/90_targeted_sequencing.sql

```diff
@@ -52,10 +52,6 @@ CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','Agilent SureSelect XT'
 CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','Agilent SureSelect XT2',1);
 CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','GA_multiplex',1);
 CALL addTargetedSequencing('SureSelect v4','GSLE Legacy','KAPA Hyper Prep',1);
-CALL addTargetedSequencing('SureSelect v5', 'FFPE-R', 'Agilent SureSelect XT2', 0);
-CALL addTargetedSequencing('SureSelect v5','GSLE Legacy','KAPA Hyper Prep',1);
-CALL addTargetedSequencing('SureSelect v5','GSLE Legacy','QXT',0);
-CALL addTargetedSequencing('SureSelect v5','FFPE-R','KAPA Hyper Prep and SureSelect XT2 v5',0);
```

Comment:
Will there be any ill effects with removing these rather than replacing `SureSelect v5` with `Agilent SureSelect Human All Exon V5`?